wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -324,14 +324,14 @@ safe) semantics.
    * Whether the command is currently grouped in a command group.  Used as extra
    * insurance to prevent accidental independent use of grouped commands.
    */
-  bool IsGrouped() const;
+  bool IsComposed() const;
```

Comment:
Does this (and SetGrouped()) need a backwards-compatible (deprecated) shim?

Reply:
It's not intended for users.

Reply:
It may not have been *intended* for users, but as a public function, users could be using it.  A deprecated shim here would be pretty easy, right?